src/services/storage/handlers/offer.ts

```diff
@@ -68,9 +76,9 @@ const handler: Handler<StorageServices> = {
 
     await offer.save()
 
-    // @TODO if we receive an unknown message then update actually doesn't happens, because offer don't changed
-    // Should we emit `update` event then???
-    if (offerService.emit) offerService.emit(created ? 'created' : 'updated', offer.toJSON())
+    if (offerService.emit && created) {
```

Comment:
You have to emit `updated` event for the `BillingPlanSet` as well no?

Reply:
Hmm, that's interesting. We are not updating offer in this case, should we send offer update event?
For me, it's not clear, if I receive offer update event but offer actually doesn't changed

Reply:
I see. Well it is true that we have Billing Plans as separate models. But they are presented as nested objects in the end point... 🤔 I guess toJSON() does not include associations.

It is true that we don't really need this information for the Pining Service, but maybe this should be included for the Marketplace UI?

Reply:
Yes, this make sense, and `toJSON ` includes associations. Then will emit it also for BillingPlans
Thanks 👍